CONTRIBUTING.md

```diff
@@ -1,94 +1,121 @@
 
-**Step 6:** Discuss the Pull Request with the .NET team; make any requested
-updates to your branch. When they are ready to accept the PR, they will add a
-"LGTM" (Looks Good To Me) comment and any other steps that are (maybe) needed.
+Place the article in the appropriate folder and any needed images in a media folder located in the same folder as the article.
 
-## Guidance checklist ##
+Be sure to follow the proper Markdown syntax.
+ 
+If you have code samples, place them in a folder within the `/samples/` folder. 
+Also do not forget to remove the "wrench" icon (🔧) from the TOC and the file heading, if applicable. 
 
-Below is a short list of guiding rules that you should keep in mind when you are
-contributing to .NET Core documentation.
+**Step 5:** Submit a PR from your branch to `dotnet/core-docs/master`.
 
-- Don't forget to submit an issue before starting work on an article
-- Don't forget to create a separate branch before working on your article
-- Don't update or `merge` your branch after you submit your pull request
-- If updating code samples in `/samples/`, be sure any line number references
-	in your article remain correct
+If your PR is addressing an existing issue, add the `Fixes #Issue_Number` keyword to the commit message or PR description, so the issue can be automatically closed when the PR is merged. For more information, see [Closing issues via commit messages](https://help.github.com/articles/closing-issues-via-commit-messages/).
 
-## Building the docs ##
+The .NET team will review your PR and let you know if the change looks good or if there are any other updates/changes necessary in order to approve it.
 
-The documentation is built using [docfx](http://dotnet.github.io/docfx/) and
-[Markdown](https://daringfireball.net/projects/markdown/syntax). It is hosted on the [.NET Core](http://dotnet.github.io/) website. 
+**Step 6:** Make any necessary updates to your branch as discussed with the team. 
 
-To build the docs, you will need to install
-[docfx](http://dotnet.github.io/docfx/); latest versions are the best. 
+When we are ready to accept the PR, we will let you know that the change looks good and merge it into the master branch. The master branch is our staging environment. 
 
-There are several ways to use docfx, and most of them are covered in the docfx [getting started guide](http://aspnet.github.io/docfx/tutorial/docfx_getting_started.html). This small guide will use the [DNX-based](http://aspnet.github.io/docfx/tutorial/docfx_getting_started.html#use-docfx-under-dnx) version of the tool to be able to invoke it from the command line; if you are comfortable with other ways listed on the link above, feel free to use those. 
+On a certain cadence, we push all commits from master branch into the live branch (our production environment) and then you'll be able to see your contribution live at https://docs.microsoft.com/dotnet/. 
```

Comment:
Don't forget to update this master/live comment if/when we change our procedure.